atm.js

```diff
@@ -1 +1,27 @@
-// your code goes here!
+var deposit1 = document.querySelector ("#checkingDepositButton")
+var withdraw1 = document.querySelector ("#checkingWithdrawalButton")
+var deposit2 = document.querySelector ("#savingsDepositButton")
+var withdraw2 = document.querySelector ("#savingsWithdrawalButton")
+
+function handleClickEvent (){
+
+  console.log("I was clicked");
+  console.log(this);
+
+}
+// This is the eventListener.
+button.addEventListener("click", handleClickEvent );
+
+
+
+//This is the eventListener.
+deposit1.addEventListener("click", handleClickEvent );
+withdraw1.addEventListener("click", handleClickEvent );
+deposit2.addEventListener("click", handleClickEvent );
+withdraw2.addEventListener("click", handleClickEvent );
+input.addEventListener("onClick" = function());
```

Comment:
The same problem here, input is not defined. Also, there is no onClick event, and the syntax is wrong, it should look the lines above.